backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -361,7 +361,7 @@ def prompt_responder(
                     doc_path=doc_path,
                     doc_name=doc_name,
                     tool=tool,
-                    prompts=prompts,
+                    prompt=prompt_instance,
```

Comment:
We were creating a list and appended only 1 entry to the list. I refactored this so that in the log message we can clearly specify the prompt that caused the issue - we execute 1 prompt at a time in prompt service for normal execution